webpack.config.babel.js

```diff
@@ -1,10 +1,67 @@
-import webpack from 'webpack';
-import ExtractTextPlugin from 'extract-text-webpack-plugin';
-import HtmlWebpackPlugin from 'html-webpack-plugin';
-import autoprefixer from 'autoprefixer';
+const webpack = require('webpack');
```

Comment:
yes I was using `"presets": [["es2015", { "modules": false }]]` then it was not able to understand `import` statement (`SyntaxError: Unexpected token import`).
somehow I forgot to commit `babelrc` :-)